FAQ.md

```diff
@@ -534,3 +534,10 @@ This means we sometimes have numbers that do connect to a real person.
 If we by chance have actually listed your real number and would like it removed,
 please report this through Google's new [Issue Tracker](http://issuetracker.google.com/issues/new?component=192347).
 Only our internal team will have access to your identity (whereas GitHub usernames are public).
+
+### Why the national number parsed has different value than input in JavaScript?
```

Comment:
Maybe better phrased as the following:
Why can the parsed national number have a different value in Javascript than what was entered?

Reply:
I think it's confusing though, because the parsed number can always have a different value. E.g. 642032345678, region NZ, would be 2032345678.

Ok a second try:

Why can the smallest digits of parsed numbers that are very long be incorrect when parsing in Javascript?

Reply:
Yupp.. this is more accurate. Just mentioning national number may not be good enough for users.
Thanks Lara.